### DISCUSSION TITLE
Frigate double-notification on iOS using SgtBatten's blueprint

### DISCUSSION THREAD

**Discussion by raspman** [blakeblackshear/frigate#15747] [Ask A Question]
Hi, I use the blueprint on 2 cameras - one camera never gives me a double notification on an event and the other camera periodically does.  Here is my config and happy to share the blueprint, too, but I don't see anything, offhand, with the blueprint, which would cause this:

`mqtt:
  host: xx
  port: xx
  user: xx
  password: xx
  topic_prefix: frigate

record:
  enabled: true
  retain:
    days: 30
    mode: all
  alerts:
    retain:
      days: 12
  detections:
    retain:
      days: 3

snapshots:
  enabled: true
  retain:
    default: 4

ffmpeg:
  hwaccel_args: preset-vaapi
  output_args:
    record: preset-record-generic-audio-copy
  input_args: preset-http-reolink

detectors:
  ov:
    type: openvino
    device: GPU

model:
  width: 300
  height: 300
  input_tensor: nhwc
  input_pixel_format: bgr
  path: /openvino-model/ssdlite_mobilenet_v2.xml
  labelmap_path: /openvino-model/coco_91cl_bkgr.txt

go2rtc:
  streams:
    garage:
      - ffmpeg:http://192.168.xx.xx/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=xx&password=xx#video=copy#audio=copy#audio=opus
    frontdoor:
      - ffmpeg:http://192.168.xx.xx/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=xx&password=x#video=copy#audio=copy#audio=opus

cameras:
  garage:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/garage
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/garage
          input_args: preset-rtsp-restream
          roles:
            - detect
    motion:
      mask:
        - 0,0,0.005,0.08,0.472,0.083,0.47,0
        - 0.279,0.02,0.945,0,0.933,0.36,0.285,0.223
    zones:
      garage_driveway:
        coordinates: 0.325,1,0.735,0.403,1,0.275,1,1
        loitering_time: 0
    objects:
      track:
        - person
    review:
      alerts:
        required_zones: garage_driveway
  frontdoor:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/frontdoor
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/frontdoor
          input_args: preset-rtsp-restream
          roles:
            - detect
    motion:
      mask: 0,0,1,0,1,0.652,0,0.623
    objects:
      track:
        - person
    zones:
      Front_door:
        coordinates: 0,0.635,1,0.659,1,1,0,1
        loitering_time: 0
        objects: person
    review:
      alerts:
        required_zones: Front_door
version: 0.15-0

semantic_search:
  enabled: true
  reindex: false
  model_size: small`

**Comment by hawkeye217** [comment]
This Github repo is for Frigate itself, you'll want to post on the sgtbatten repo for help with the blueprint.

**Reply by raspman** [reply]
I tried that - he didn't see any issues, so I thought I would check on this side.  Because there is some configuration on the frigate side (e.g., alerts: required_zones: Front_door) I thought I would check over here. I know it's something simple, just no idea what it is.

**Reply by hawkeye217** [reply]
Your frigate config looks fine. You could watch the MQTT topics (/review or /events) that the blueprint uses to perhaps pinpoint the issue further.

**Reply by raspman** [reply]
I found that area, but what would I put in here?
![Screenshot 2024-12-30 at 7 11 58 PM](https://github.com/user-attachments/assets/51a45368-03c0-4759-b859-37639be948a5)

**Reply by hawkeye217** [reply]
`frigate/reviews`

https://docs.frigate.video/integrations/mqtt#frigatereviews

**Reply by raspman** [reply]
That helps, thanks.
